dir-index-uncat.html

```diff
@@ -2,6 +2,21 @@
 <html>
 <head>
   <meta charset="utf-8" />
+  <meta name="description" content="A directory of files hosted on the distributed, decentralized web using IPFS">
+  <meta property="og:title" content="Files on IPFS">
+  <meta property="og:description" content="A directory of files hosted on the distributed, decentralized web using IPFS">
+  <meta property="og:type" content="website">
+  <meta property="og:image" content="https://gateway.ipfs.io/ipfs/QmSDeYAe9mga6NdTozAZuyGL3Q1XjsLtvX28XFxJH8oPjq">
+  <meta name="twitter:title" content="Files on IPFS">
+  <meta name="twitter:description" content="A directory of files hosted on the distributed, decentralized web using IPFS">
+  <meta name="twitter:card" content="summary_large_image">
+  <meta name="twitter:image" content="https://gateway.ipfs.io/ipfs/QmSDeYAe9mga6NdTozAZuyGL3Q1XjsLtvX28XFxJH8oPjq">
+  <meta name="twitter:creator" content="@ipfs">
+  <meta name="twitter:site" content="@ipfs">
+  <meta name="image" content="https://gateway.ipfs.io/ipfs/QmSDeYAe9mga6NdTozAZuyGL3Q1XjsLtvX28XFxJH8oPjq">
+  <meta name="keywords" content="IPFS, dweb, protocol, libp2p, ipld, multiformats, bitswap, decentralized web, InterPlanetary File System, dapp, Protocol Labs">
```

Comment:
I worry these keywords will do more harm than good: as google bots crawl public gateways and see it everywhere, seacher engines will quickly learn to ignore them and perhaps even mark them as "spammy words" and decrease SEO juice. I'd say remove it (better safe than sorry).

```suggestion
```

Reply:
Fair enough. Those are the same keywords we are using in IPFS docs, fwiw, but at the sort of scale we might be seeing for dir-index-html instantiations, that could potentially be a concern.